test.rb

```diff
@@ -0,0 +1,5 @@
+if foo
+  foo.bar
+end
+
+foo bar hoge fuga;
```

Comment:
__[development][rubocop]__
* Do not use semicolons to terminate expressions. [:ref](https://github.com/bbatsov/ruby-style-guide#no-semicolon)

<!-- Comment by [SideCI](https://sideci.com) -->